wordstats/testquiz.py

```diff
@@ -0,0 +1,18 @@
+from wordstats.cognate_info import CognateInfo
+from wordstats.loading_from_hermit import load_language_from_hermit
+from wordstats.word_distance import WordDistance
+
+german = list(load_language_from_hermit("de").word_info_dict.keys())
+dutch = list(load_language_from_hermit("nl").word_info_dict.keys())
+
+cognateFM = CognateInfo("denl", "edit_distance_rules")
+
+distanceMetric = WordDistance.loadConfig("denl", "edit_distance_rules")
+distanceMetric.load_rules("denl", "edit_distance_rules")
+
+cognateFM.apply_distance_metric(german, dutch, distanceMetric.edit_distance_rules)
+cognateFM.save_candidates()
+
+print(cognateFM.candidates)
+cognateFM.start_quiz()
+cognateFM.save_evaluation()
```

Comment:
hmm. 
if for an evaluation there are 3000 candidates, then
the only way the info is saved is if somebody runs the program and provides feedback for all the 3000.
if by mistake the program stops for example, nothing is saved. 
i think the info should be saved as soon as somebody provides feedback on a given cognate. 

this also explains my comment on the start_quiz() method.